setup.py

```diff
@@ -106,7 +106,7 @@ def run(self):
         'theme.conf',
         '*.html',
         'static/css/*.css',
-        'static/css/fonts/*.*'
+        'static/css/fonts/*.*',
```

Comment:
Woops! Was this affecting user installations?

Reply:
It does not seem like it